.travis.yml

```diff
@@ -1,26 +1,23 @@
+sudo: required
+dist: trusty
 language: cpp
 
 # NOTE: The COMPILER variable is unused. It simply makes the display on
 # travis-ci.org more readable.
 matrix:
     include:
         - compiler: gcc
-          env: COMPILER=g++-4.6     STD=c++0x BUILD_TYPE=Coverage
-        - compiler: gcc
-          env: COMPILER=g++-4.6     STD=c++0x BUILD_TYPE=Debug
-        - compiler: gcc
-          env: COMPILER=g++-4.6     STD=c++0x BUILD_TYPE=Release
+          env: COMPILER=g++         STD=c++11 BUILD_TYPE=Debug
         - compiler: gcc
-          env: COMPILER=g++-4.8     STD=c++11 BUILD_TYPE=Debug
+          env: COMPILER=g++         STD=c++11 BUILD_TYPE=Debug
```

Comment:
sorry, by "above" i meant within this file.

lines 9-10 and 11-12 are identical. right? can we remove one stanza?